src/Nethermind/Nethermind.Blockchain.Test/Validators/ShardBlobBlockValidatorTests.cs

```diff
@@ -35,7 +38,10 @@ public static bool Blob_gas_fields_should_be_set(IReleaseSpec spec, ulong? blobG
     [TestCaseSource(nameof(BlobsPerBlockCountTestCases))]
     public bool Blobs_per_block_count_is_valid(ulong blobGasUsed)
     {
-        ISpecProvider specProvider = new CustomSpecProvider(((ForkActivation)0, Cancun.Instance));
+        ISpecProvider specProvider = new OverridableSpecProvider(
+            new CustomSpecProvider(((ForkActivation)0, Cancun.Instance)),
+            r => new OverridableReleaseSpec(r) { MaxBlobCount = MaxBlobCount, TargetBlobCount = TargetBlobCount });
```

Comment:
why do we need to do this? just set these default values in Cancun release spec?